packages/atlas/src/components/_overlays/Dialog/Dialog.styles.ts

```diff
@@ -81,25 +92,27 @@ const getDenseHeaderContentStyles = ({ denseHeader, noContentPadding }: ContentP
 export const Content = styled.div<ContentProps>`
   overflow-y: auto;
   overflow-x: hidden;
+  display: grid;
   padding: ${({ noContentPadding }) => !noContentPadding && 'var(--local-size-dialog-padding)'};
   ${getDenseHeaderContentStyles};
 `
 
 export const footerDividersStyles = css`
-  box-shadow: inset 0 1px 0 0 ${oldColors.gray[600]};
+  box-shadow: ${cVar('effectDividersTop')};
   padding-top: var(--local-size-dialog-padding);
 `
 
 type FooterProps = {
-  hasAdditionalActions: boolean
-  additionalActionsNodeMobilePosition: 'bottom' | 'top'
+  'data-hasadditionalactions'?: boolean
```

Comment:
```suggestion
  'data-has-additional-actions'?: boolean
```
This is not really readable